.markdownlint-cli2.jsonc

```diff
@@ -0,0 +1,126 @@
+{
+  "config": {
+    "default": true,
+    // MD001 - Heading levels should only increment by one level at a time.
+    // -> Disabled, as some callouts include headings.
+    "MD001": false,
+    "MD004": {
+      "style": "dash"
+    },
+    "MD007": {
+      "indent": 2
+    },
+    "MD010": {
+      "spaces_per_tab": 2
+    },
+    "MD013": false,
+    // Fixed upstream, needs PRs in this repo
+    "MD014": false,
+    // Fixed upstream, needs PRs in this repo
+    "MD023": false,
+    // Fixed upstream with "allow_different_nesting": true
+    "MD024": false,
+    "MD025": {
+      "front_matter_title": "^\\s*title\\s*[:=]"
+    },
+    "MD026": false,
+    // Consectutive Notes/Callouts currently don't conform with this rule
+    "MD028": false,
+    "MD033": {
+      "allowed_elements": [
+        "a",
+        "abbr",
+        "annotation",
+        "base",
+        "br",
+        "caption",
+        "code",
+        "col",
+        "colgroup",
+        "dd",
+        "details",
+        "dfn",
+        "div",
+        "dl",
+        "dt",
+        "em",
+        "h4",
+        "h5",
+        "img",
+        "kbd",
+        "li",
+        "math",
+        "menclose",
+        "mfenced",
+        "mfrac",
+        "mfrac",
+        "mi",
+        "mmultiscripts",
+        "mn",
+        "mo",
+        "mover",
+        "mphantom",
+        "mprescripts",
+        "mroot",
+        "mrow",
+        "ms",
+        "mspace",
+        "mspace",
+        "msqrt",
+        "mstyle",
+        "msub",
+        "msubsup",
+        "msup",
+        "mtable",
+        "mtd",
+        "mtext",
+        "mtr",
+        "munder",
+        "munderover",
+        "none",
+        "ol",
+        "p",
+        "pre",
+        "section",
+        "semantics",
+        "strong",
+        "sub",
+        "summary",
+        "sup",
+        "table",
+        "tbody",
+        "td",
+        "tfoot",
+        "th",
+        "thead",
+        "tr",
+        "ul",
+        "var"
+      ]
+    },
+    // Fixed upstream, needs PRs in this repo
+    "MD034": false,
+    // Fixed upstream, needs PRs in this repo
+    "MD036": false,
+    // Upstream fixing in https://github.com/mdn/content/pull/20115
+    "MD037": false,
+    "MD040": false,
+    // See https://github.com/mdn/content/pull/20026, as macros currently break this
+    "MD042": false,
+    // Tracked upstream as part of https://github.com/mdn/content/pull/19334
+    "MD045": false,
+    "MD046": {
+      "style": "fenced"
+    },
+    // MD049 is set to underscore upstream, not yet fixed here
+    "MD049": false,
+    // MD050 is set to asterisk upstream, not yet fixed here
+    "MD050": false,
+    // MD051 - Link fragments should be valid.
+    // -> Disabled, as yari generates link fragments by replacing spaces with underscores, not dashes.
+    "MD051": false,
+    // Incompatible with brackets in macros like MD042
+    "MD052": false
+  },
+  "ignores": ["node_modules"]
+}
```

Comment:
Missing trailing newline

Reply:
Thanks, copied the `.editorconfig` from mdn/content and resaved